### PR TITLE
[RAUI-7] Reusable Tabbed View Component

### PR DESCRIPTION
**Pull request by beingtmk**

### CODE REVIEW

**Comment by mogoodrich** [comment]
Thanks @beingtmk , I will take a look in the next couple of days.

File: src/components/tabs/Tabs.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Tab from './Tab';
+
+import '../../../assets/css/tabs.css';
+
+class Tabs extends Component {
+  static propTypes = {
+    children: PropTypes.instanceOf(Array).isRequired,
```
Comment (Abdo-A) [inline src/components/tabs/Tabs.jsx]: Best propTypes for 'children' is:
```
PropTypes.oneOfType([
        PropTypes.arrayOf(PropTypes.node),
        PropTypes.node
    ]).isRequired
```

Comment (beingtmk) [reply src/components/tabs/Tabs.jsx]: Thank you for pointing it out, i will change it!

File: src/components/tabs/Tab.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Tab extends Component {
+  static propTypes = {
```
[the remark this answers is not in the document]
Comment (beingtmk) [reply src/components/tabs/Tab.jsx]: Thank you, i will update it accordingly!

**Comment by mogoodrich** [comment]
Thanks @beingtmk !  And thanks @rhenshaw56  and @Abdo-A ....

@rhenshaw56  or @jomadoye  if you have time, it would be great to pull this PR into your local build and test that the component works as expected.  Thoughts?

File: package-lock.json [hunk 3]
```diff
@@ -7499,7 +7500,8 @@
           "version": "2.1.1",
           "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-2.1.1.tgz",
           "integrity": "sha1-w7M6te42DYbg5ijwRorn7yfWVN8=",
-          "dev": true
+          "dev": true,
+          "optional": true
```
Comment (mogoodrich) [inline package-lock.json]: Do you know what all these "optional" additions to the package-lock mean?

Comment (beingtmk) [reply package-lock.json]: searching around in google seems to suggest that came from me install from newer version of npm
[Stack Overflow](https://stackoverflow.com/questions/54411377/package-lock-json-file-package-with-optional-true)

**Comment by mogoodrich** [comment]
Thanks @beingtmk , let me know if you've had a chance to address the other comments. It also looks like there are conflicts now that need to be resolved.

**Comment by beingtmk** [comment]
I am on it!

File: package-lock.json (hunk 3 above)
Comment (beingtmk) [reply package-lock.json]: should i remove them?

Comment (mogoodrich) [reply package-lock.json]: Fine to keep them, we can revert if there are any issues!

**Comment by mogoodrich** [comment]
Looks good, I just merged, thanks @beingtmk !

**Pull request merged**